python/ray/utils.py

```diff
@@ -282,17 +282,6 @@ def setup_logger(logging_level, logging_format):
     logger.propagate = False
 
 
-def try_update_handler(new_stream):
```

Comment:
@richardliaw please take a look, this no longer seems necessary.

Reply:
oh because dup2 -- yeah this is probably fine; this was basically to pass one of the tests in `run_test` so as long as this passes it's fine